vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -111,26 +111,33 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 			var errno syscall.Errno
 			errno = err.(syscall.Errno)
 			msg = fmt.Sprintf("Run '%s' failed: %v (errno=%d).", cmd, err, int(errno))
-
-			// Still below maximum number of retries, log and continue
+			str := C.GoString(ans.errBufP)
```

Comment:
that's slightly weird. ` &ans.errBuf[0]` is literally ans.errBuf value in C.  What does not work if you simply use C.goString(ans.errBuf) ?  Even though char[] amd char* can provide different location for initialized content, the actual name means the same address in C...